src/OmnichannelChatSDK.ts

```diff
@@ -2182,6 +2263,11 @@ class OmnichannelChatSDK {
                 throw error // Bubble up error by default to throw ChatConfigRetrievalFailure
             }
         }
+        return this.liveChatConfig;
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    private async buildConfigurations(liveChatConfig: any) {
```

Comment:
Can this one be typed?

Reply:
this one is used only internally, and it takes the object as is defined from OCSDK, which defines the return as   
`public async getChatConfig(requestId: string, bypassCache = false): Promise<object | void>`